src/main/scala/com/gu/automation/core/BaseFeatureSpec.scala

```diff
@@ -15,12 +18,17 @@ abstract class BaseFeatureSpec[T <: WebDriver] extends fixture.FeatureSpec with
       val browserEnhancedSpecText = s"$specText on $browser"
       scenario(browserEnhancedSpecText, testTags: _*)({ td =>
         sys.props.put("teststash.url", Config().getPluginValue("teststash.url"))
+        val setName = Config().getProjectName()
+        val setDate = BaseFeatureSpec.startTime.getMillis.toString
         MDC.put("ID", UUID.randomUUID().toString)
-        MDC.put("setName", Config().getProjectName())
-        MDC.put("setDate", BaseFeatureSpec.startTime.getMillis.toString)
+        MDC.put("setName", setName)
+        MDC.put("setDate", setDate)
         MDC.put("testName", td.name)
         MDC.put("testDate", DateTime.now.getMillis.toString)
         MDC.put("phase", "STEP")
+        val tstashName = URLEncoder.encode(setName, "UTF-8")
+        val tstashURL = s"http://54.77.196.196:9000/setLookup?setName=$tstashName&setDate=$setDate"
```

Comment:
is the tstash base url in the config file already?  might be worth using that instead of hard coding.